docs/testing.rst

```diff
@@ -0,0 +1,140 @@
+Testing
+=======
+
+When building Flask apps that have integrated Flask-Dance you'll eventually run
+into the need to test your app. But if routes are guarded by the requirement of
+having a valid oauth2 session, i.e getting passed the `.authorized()` call on
```

Comment:
"OAuth" should be capitalized. You're using "passed" instead of "past" -- the correct spelling should be "getting past". Also, fixed-width text requires two sets of backticks in rst: 

    ``.authorized()``